Dockerfile

```diff
@@ -1,33 +1,36 @@
-FROM alpine
+FROM java:openjdk-8
 
-RUN apk add --update \
-	bash \
+# Download a reference binary that tests can run against.
+ENV REFERENCE_VERSION="beta-20160407"
```

Comment:
I recommend putting this at the end, right before the curl download, because otherwise changing this version will invalidate the entire cache and we have to run the entire apt-get install again.